lib/oli/interop/ingest.ex

```diff
@@ -718,14 +735,24 @@ defmodule Oli.Interop.Ingest do
             |> Enum.filter(fn f -> !is_nil(f) end)
         },
         resource_type_id: Oli.Resources.ResourceType.id_for_page(),
-        scoring_strategy_id: Oli.Resources.ScoringStrategy.get_id_by_type("average"),
+        scoring_strategy_id:
+          Map.get(page, "scoringStrategyId", Oli.Resources.ScoringStrategy.get_id_by_type("best")),
         graded: graded,
-        max_attempts:
-          if graded do
-            5
-          else
-            0
-          end
+        relates_to:
+          Map.get(page, "relatesTo", []) |> Enum.map(fn id -> String.to_integer(id) end),
+        max_attempts: Map.get(page, "maxAttempts", if(graded, do: 5, else: 0)),
+        explanation_strategy:
+          Map.get(page, "explanationStrategy", graded)
+          |> get_explanation_strategy(),
```

Comment:
Is "graded" (in Map.get) a correct default for the explanation strategy?

Reply:
Nil is default

Reply:
If the value is from (Map.get) is actually `nil` it will return a `nil` and not "graded"